docs-sandbox/index.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Substrate empowers developers
```

Comment:
```suggestion
title: Substrate Empowers Developers
```
Page titles in title case?

Reply:
Don't think so, no see: https://docs.substrate.io/v3/contribute/writing/#punctuation on capitalization.

Reply:
> All heading levels should use the following conventions: Use sentence style case.

Headings are not Titles.... are they?

Reply:
All heading levels and titles should use sentence style case.